### ISSUE TITLE
Unable to set Executable or Steam path when starting a 2nd preset without saving the 1st

### ISSUE THREAD

**Issue by kencinder** [SteamGridDB/steam-rom-manager#111]
Not sure exactly how to describe this better than the title...

If you have a preset with changes made to it, and it's icon is yellow, creating a new preset will not allow you to set the `Executable` or `Steam Directory` on the new preset. It stays empty stating the path is invalid until you go back to the other parser and save it.

**Comment by FrogTheFrog** [comment]
I can't replicate it :/. Could you post specific steps?

**Comment by kencinder** [comment]
Actually it seems it's due to selecting a preset, setting the path to executable, and then starting over with another preset. It had appeared unsaved preset was causing it...doesn't seem to be the case.

1: Select a preset
2: Set `Executable` path
3: Select a new preset
4: Set `Executable` path - Failure

Not sure why undoing or saving changes appeared to correct it, However selecting a view that isn't a parser, seems to reset the behaviour. ie; Selecting `Settings` or `About`

**Comment by FrogTheFrog** [comment]
I still can't replicate that :/

**Comment by ghost** [comment]
Replicated. Exclusively with the `Executable` path (Steam Directory path sets just fine.)

Edit: FULLY replicated, BOTH paths affected.

**Comment by ChibiChoko** [comment]
I tested this as well following Ken's instructions (starting from a blank parser), and replicated the issue perfectly.

**Comment by HEspoke** [comment]
I played around with this bug for a little while. I confirmed the bug as well and I was trying to figure out how a user would actually encounter the bug. 
If a user selected a preset config, set the above mentioned paths, and then changed their mind and selected a new preset config the user would be unable to change the paths until navigating from the Parser option or closing SRM and starting over.
This bug was a very good find kencinder . The likelihood of a user encountering this is rather small but that doesn't diminish its importance.
